### ISSUE TITLE
💬 RFC: redundant API catalog views

### ISSUE THREAD

**Issue by pjungermann** [backstage/backstage#18457]
### 🔖 Need

At the moment, we have two catalog views for the API entity:

1.  the standard catalog view filtered by kind API: https://demo.backstage.io/catalog?filters%5Bkind%5D=api&filters%5Buser%5D=all
![Screenshot 2023-06-26 at 21 50 04](https://github.com/backstage/backstage/assets/2513586/11cc2104-8409-4f14-b198-b90010da0830)

2. the api-docs view, provided by `@backstage/plugin-api-docs`
![Screenshot 2023-06-26 at 21 50 45](https://github.com/backstage/backstage/assets/2513586/66b5ed89-c1d4-4660-b48f-4f377b742386)

Both views have minor differences:

- header with title and sub-title
- kind select field only in the catalog view
- processing status filter only available at the catalog view
- different default filter (all vs owned)

### 🎉 Proposal

Drop

- route `/api-docs`
- the nav entry
- the page behind it (`ApiExplorerPage`)

at Backstage and only provide one view going forward instead of two similar/redundant views. Less confusion.

### 〽️ Alternatives

All adapters can decide this on their own.

### 👀 Have you spent some time to check if this RFC has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

**Comment by benjdlambert** [comment]
Hmm, so I think this opens up another discussion with what the `/catalog` page will eventually become. Internally at Spotify we don't even have this page at all and it's not in the developers path for their daily use. We have a more specialised manage page which just shows you all the things that you own and care about, but of course you can use search to find other components by name.

That being said, that doesn't mean that this doesn't provide value to other end users and companies - I would like to suggest an approach that maybe the `api-docs` plugin doesn't just resurface the same data in the same view. I think that's what makes it a little bit unfortunate. Take the scaffolder as an example, look how you can see Templates in the /catalog view, but actually when you're in the `/scaffolder` route, you see these things as a different experience. It's much more rich and shows you information which you care about, but with similar search and filtering as the `catalog` view.

It's also arguable that even that is too close to the catalog view with the filters etc, but at least I think it's on the right track.

Thoughts?

**Comment by awanlin** [comment]
I agree, it is confusing. The nice thing is that right now you have some control over this. In our case we just deleted the APIs option from the sidebar

**Comment by OrkoHunter** [comment]
Very similar to TechDocs' issue as well https://github.com/backstage/backstage/issues/3122

I'd say just like @awanlin, folks can remove the API sidebar item and not expose the two views to their developers.

**Comment by hntrdglss** [comment]
Related #19911
